### PR TITLE
updated the necessary docs for ethers

### PR DESCRIPTION
**Pull request by codebuster22**

### CODE REVIEW

**Review by dkent600 — DISMISSED** [review]

File: scripts/gnosis.js [hunk 1]
```diff
@@ -3,38 +3,32 @@
...
-        }
+        payload
     );
-    return await res.json();
+    return res;
 }
 
 const getCurrentNonce = async (safe) => {
     const transactions = await get('getNonce', safe);
+    // console.log(transactions);
```
Comment (kinspotting) [inline scripts/gnosis.js:31]: pls delete the comment

File: docs/dapp-integration.md [hunk 2]
```diff
@@ -412,8 +412,8 @@ transaction.nonce = await gnosis.getCurrentNonce();
 
 6. Call(), send details to Signer contract to generate hash and sign the hash.
 ```js
-// I'm confused with the equivalent of this in ethersjs
-const {hash, signature} = await signer.methods.generateSignature(
+// I hope it returns an object
```
Comment (kinspotting) [inline docs/dapp-integration.md:415]: pls change the comment to "should return an object"

**Review by kinspotting — CHANGES_REQUESTED** [review]
needs minor changes

File: scripts/gnosis.js [hunk 3]
```diff
@@ -3,34 +3,27 @@
...
+    return res;
 }
 
 const get = async (method, safe) => {
-    const res = await fetch(
+    const res = await axios.get(
         `${url}${safe}${methods[method]}`
     );
-    return await res.json();
+    // console.log(res.data);
```
Comment (kinspotting) [inline scripts/gnosis.js:17]: pls delete the comment

File: scripts/change_owner.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 require('dotenv').config({path:'./.env'});
 const hre = require('hardhat');
 const {PROVIDER_KEY, MNEMONIC} = process.env;
-const {'4': {SeedFactory: SEED_FACTORY, Safe}} = require('../contract-addresses.json');
+const {'rinkeby': {SeedFactory: SEED_FACTORY, Safe}} = require('../contractAddresses.json');
```
Comment (kinspotting) [inline scripts/change_owner.js:4]: let's not hardcode the network name in the script, network name could be stored in the `.env` file, pls reference to the `contracts` repo

**Review by kinspotting — CHANGES_REQUESTED** [review]

**Review by kinspotting — APPROVED** [review]
lgtm

**Pull request merged**